backend/src/test/java/de/bund/digitalservice/ris/adm_vwv/application/converter/LdmlConverterServiceTest.java

```diff
@@ -426,6 +427,40 @@ void convertToBusinessModel_documentTypeZusatz() {
       .isEqualTo("Bekanntmachung");
   }
 
+  @Test
+  void convertToBusinessModel_normReference() {
+    // given
+    String xml = TestFile.readFileToString("ldml-example.akn.xml");
+    DocumentationUnit documentationUnit = new DocumentationUnit(
+      "KSNR20250000001",
+      UUID.randomUUID(),
+      null,
+      xml
+    );
+
+    // when
+    DocumentationUnitContent documentationUnitContent = ldmlConverterService.convertToBusinessModel(
+      documentationUnit
+    );
+
+    // then
+    assertThat(
+      documentationUnitContent.normReferences().get(0).normAbbreviation().abbreviation()
+    ).isEqualTo("PhanGB");
+    assertThat(
+      documentationUnitContent.normReferences().get(0).normAbbreviation().officialLongTitle()
+    ).isEqualTo("PhanGB § 1a Abs 1");
+    assertThat(
+      documentationUnitContent.normReferences().get(0).singleNorms().get(0).singleNorm()
+    ).isEqualTo("§ 1a Abs 1");
+    assertThat(
+      documentationUnitContent.normReferences().get(0).singleNorms().get(0).dateOfVersion()
+    ).isEqualTo("2022-02-02");
+    assertThat(
+      documentationUnitContent.normReferences().get(0).singleNorms().get(0).dateOfRelevance()
+    ).isEqualTo("2011");
```

Comment:
Just a suggestion to make it more compact:
```suggestion
assertThat(documentationUnitContent.normReferences())
      .hasSize(2)
      .first()
      .extracting(
        nr -> nr.normAbbreviation().abbreviation(),
        nr -> nr.normAbbreviation().officialLongTitle(),
        nr -> nr.singleNorms().getFirst().singleNorm(),
        nr -> nr.singleNorms().getFirst().dateOfVersion(),
        nr -> nr.singleNorms().getFirst().dateOfRelevance()
      )
      .containsExactly("PhanGB", "PhanGB § 1a Abs 1", "§ 1a Abs 1", "2022-02-02", "2011");
```

Reply:
:+1: Learning about the "streaming" way of transforming/extracting is something I really enjoy. Please keep it coming!

Unfortunately it seems like each "library" (here: assertions, Lists; but same for Optionals and Streams) uses similar, but slightly different syntax for their collections, e.g. `.first()` vs `.getFirst()`.